main/config.c

```diff
@@ -255,7 +255,16 @@ struct ast_config {
 	struct ast_category *last_browse;     /*!< used to cache the last category supplied via category_browse */
 	int include_level;
 	int max_include_level;
-	struct ast_config_include *includes;  /*!< a list of inclusions, which should describe the entire tree */
+	/*! First inclusion in the list. */
+	struct ast_config_include *includes_root;
+	/*! Last inclusion in the list. */
+	struct ast_config_include *includes_last;
+};
+
+struct ast_config_included_file {
```

Comment:
Avoid the `ast_` prefix when creating new things that will not be exported.

Reply:
Poof.  The `ast_` prefix has been removed